src/interface/TWCoinTypeConfiguration.cpp

```diff
@@ -304,4 +304,48 @@ TWString *_Nonnull TWCoinTypeConfigurationGetName(enum TWCoinType type) {
     return TWStringCreateWithUTF8Bytes(string.c_str());
 }
 
+TWString *_Nullable TWCoinTypeConfigurationGetNodeSource(enum TWCoinType type) {
+    string string;
+    switch (type) {
+    case TWCoinTypeBitcoin:
+    case TWCoinTypeBitcoinCash:
+    case TWCoinTypeDash:
+    case TWCoinTypeDogecoin:
+    case TWCoinTypeDecred:
+    case TWCoinTypeLitecoin:
+    case TWCoinTypeGroestlcoin:
+    case TWCoinTypeZcash:
+    case TWCoinTypeZcoin:
+    case TWCoinTypeViacoin:
+    case TWCoinTypeLux:
+    case TWCoinTypeQtum: string = "https://github.com/trezor/blockbook"; break;
+    case TWCoinTypeICON: string = "https://github.com/icon-project/icon-rpc-server"; break;
+    case TWCoinTypeNEO: string = "https://github.com/neo-project/neo"; break;
+    case TWCoinTypeStellar: string = "https://github.com/stellar/go"; break;
+    case TWCoinTypeEthereum:
```

Comment:
Attention. For all cases below result will be: "https://github.com/ethereum/go-ethereum".
Need add break or empty string